packages/menu/src/SubMenu.tsx

```diff
@@ -251,6 +252,10 @@ const SubMenu = React.forwardRef((props: SubMenuProps, ref) => {
       ),
     });
 
+  const caretIconStyles = cx(open ? openIconStyle : closedIconStyle, {
```

Comment:
For cleanliness, it's probably best to make this ternary its own variable before using it here.

Reply:
👍